static/app/components/profiling/ProfilingOnboarding/util.ts

```diff
@@ -47,6 +47,12 @@ export const supportedPlatformExpectedDocKeys: Record<
     '2-configure-performance',
     '3-configure-profiling',
   ],
+  'javascript-sveltekit': [
+    '0-alert',
```

Comment:
have we added docs for the onboarding in our [sentry-docs](https://github.com/getsentry/sentry-docs) repo as well? otherwise this will be broken

Reply:
I wasn't sure if this would cause breakage or just not work for now. But regardless, I think this should be okay once https://github.com/getsentry/sentry-docs/pull/6867 is merged